cmd/lint.go

```diff
@@ -250,6 +261,7 @@ func GetLintCommand() *cobra.Command {
 	cmd.Flags().StringP("fail-severity", "n", model.SeverityError, "Results of this level or above will trigger a failure exit code")
 	cmd.Flags().Bool("ignore-array-circle-ref", false, "Ignore circular array references")
 	cmd.Flags().Bool("ignore-polymorph-circle-ref", false, "Ignore circular polymorphic references")
+	cmd.PersistentFlags().String("globbed-files", "", "Glob pattern of files to lint")
```

Comment:
Why `PersistentFlags` ?

Reply:
Should just be a flag as it only applied to the lint command for now, oops!